docs/licensing.md

```diff
@@ -1,9 +1,364 @@
 # Licensing
 
+AdonisFX requires a successful license activation for commercial use. For non-commercial and testing purposes a trial period of 30 days can be requested.
+In this page the different licensing components of AdonisFX will be explained in combination with the requirements and steps for activating the product.
+
+AdonisFX supports two *Licensing Modes*:
+
+- **Node-Locked Licensing:** License mode that locks AdonisFX to one particular computer with a particular hardware footprint. Using the license key on a different computer would disable the ability to activate the software.
+- **Floating Licensing:** License mode that allows different users to request licenses from a common license pool (served by a server) avoiding so the use restriction of the software to a single machine.
+
+AdonisFX also distinguishes between two *Licensing Types*:
+
+- **Interactive:** License type that allows the user to use AdonisFX from the graphical interface of the software where AdonisFX was loaded. This licensing type is intended for users that want to build scenes, manipulate and interact with AdonisFX using visual and interactive feedback.
+- **Batch:** License type that allows the user to use AdonisFX from a batch script or terminal without the ability to use the graphical interface for manipulating the software. This licensing type is intended for users that want to run AdonisFX from a terminal to for example render a scene on the farm after setting the scene up using an Interactive License.
+
+To be able to activate AdonisFX it is required to purchase a `PRODUCT KEY`. Product keys can be purchased through Inbibo's official website [https://www.inbibo.co.uk/](https://www.inbibo.co.uk/). AdonisFX product keys have the following characteristics:
```

Comment:
What will the purchase address be for the licenses?

Reply:
We don't have the final link yet. Add a "PENDING" for now

Reply:
Done!